### PR TITLE
Certona ca

### PR DESCRIPTION
**Pull request by rvrita**
Added CA Certona 3x1 and have a small fix for our US Certona 3x1 (the images didn't have height and width and it caused issues sometimes).

### CODE REVIEW

File: templates/templateC3x1CA.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          </td>
+        </tr>
+      </table>
+    </td>
+    <td valign="top" align="center">
+      <table width="190" border="0" cellspacing="0" cellpadding="0" align="center">
+        <tr>
+          <td align="center" valign="top">
+            <a href="[@trackurl LinkID='88a788d0e943449794cf9de410f2141e' LinkName='certonapick3' LinkTag='pl-p7' Tracked='ON' Encode='ON' AppendSuffix='ON' Render='ON' LinkType='REDIRECT']http://ris.res-x.com/?appid=Sephora02&index=3&userid=\${Profile.atg1_id!'0000'}&scheme=${tag}&campaignid=\${deploymentName}[/@trackurl]" target="_blank">
+              <img border="0" src="http://ris.res-x.com/image.aspx?appid=Sephora02&userid=\${Profile.atg1_id!'0000'}&index=3&scheme=${tag}&campaignid=\${deploymentName}" alt="" />
```
Comment (choiwa) [inline templates/templateC3x1CA.js]: @rvrita the images run larger than 700px total. can we width="190" height="306" so they will display same as the US?

**Review by choiwa — CHANGES_REQUESTED** [review]
please review the comment regarding the width and height. Thank you!

File: templates/templateC3x1CA.js (hunk 1 above)
Comment (rvrita) [reply templates/templateC3x1CA.js]: Thanks Joyce, added the image sizes

**Review by choiwa — APPROVED** [review]
looks great! Thank you.

**Pull request merged**